src/entities/reaction.ts

```diff
@@ -29,8 +30,12 @@ export class Reaction extends BaseEntity {
   @Column({ nullable: true })
   projectUpdateId: number;
 
+  @Field(type => User, { nullable: true })
+  @ManyToOne(type => User, { nullable: true })
+  user: User;
   @Field(type => ID)
   @Column()
+  @RelationId((reaction: Reaction) => reaction.user)
```

Comment:
As much as I know this is the right way, this will most likely delete the data from the UserId column, by running a query on relationId to set the column. 

Confirm this isn't the case and we can proceed.

Reply:
Else we will have to do a different query.

Reply:
Rest assured... leftJoins work without the need of relationId or ManyToOne, they are a bit less efficient because they don't have an index. But I think we can add it separately (haven't test this).
```
      .leftJoin(User, 'user', 'reaction.userId = user.id')
```

Reply:
Oh nice catch, thank you I rewrite that function considering all your feedback, it works now